polkadot/node/core/provisioner/src/lib.rs

```diff
@@ -745,24 +840,6 @@ async fn select_candidates(
 	gum::trace!(target: LOG_TARGET, leaf_hash=?relay_parent,
 				"Got {} backed candidates", candidates.len());
 
-	// `selected_candidates` is generated in ascending order by core index, and
-	// `GetBackedCandidates` _should_ preserve that property, but let's just make sure.
-	//
-	// We can't easily map from `BackedCandidate` to `core_idx`, but we know that every selected
-	// candidate maps to either 0 or 1 backed candidate, and the hashes correspond. Therefore, by
-	// checking them in order, we can ensure that the backed candidates are also in order.
-	let mut backed_idx = 0;
-	for selected in selected_candidates {
```

Comment:
This dropping of the ordering requirement is ok because of the already merged runtime changes - correct?

Reply:
no, but thanks to the sort_by here: https://github.com/paritytech/polkadot-sdk/blob/6f81a4a09270a5930b8786e2dc765347078b4d4a/polkadot/runtime/parachains/src/paras_inherent/mod.rs#L1023